apps/wallet-mobile/src/i18n/locales/en-US.json

```
@@ -8,6 +8,17 @@
   "menu.supportTitle": "Any questions?",
   "menu.supportLink": "Ask our support team",
   "menu.knowledgeBase": "Knowledge base",
+  "components.analytics.accept": "Accept",
+  "components.analytics.anonymous": "Anonymous analytics data",
+  "components.analytics.description": "Share user insights to help us fine tune Yoroi to better serve your needs.",
+  "components.analytics.header": "Join the journey to improve Yoroi",
+  "components.analytics.more": "Learn more about user insights",
+  "components.analytics.noip": "We <b>are not</b> recording IP addresses",
+  "components.analytics.nosell": "We <b>do not</b> sell data",
+  "components.analytics.optout": "You can always opt-out via Settings",
+  "components.analytics.private": "We <b>cannot</b> access private keys",
+  "components.analytics.skip": "Skip",
+  "components.analytics.toggle": "Allow Yoroi analytics",
```

we are dropping the `components` it was legacy and it doesn't follow the current best practices.
```suggestion
  "analytics.accept": "Accept",
  "analytics.anonymous": "Anonymous analytics data",
  "analytics.description": "Share user insights to help us fine tune Yoroi to better serve your needs.",
  "analytics.header": "Join the journey to improve Yoroi",
  "analytics.more": "Learn more about user insights",
  "analytics.noip": "We <b>are not</b> recording IP addresses",
  "analytics.nosell": "We <b>do not</b> sell data",
  "analytics.optout": "You can always opt-out via Settings",
  "analytics.private": "We <b>cannot</b> access private keys",
  "analytics.skip": "Skip",
  "analytics.toggle": "Allow Yoroi analytics",
```